tests/threads_nested_attach_daemon.rs

```diff
@@ -45,7 +45,5 @@ pub fn nested_attaches_should_not_detach_daemon_thread() {
         assert_eq!(jvm().threads_attached(), 1);
     }
     assert_eq!(jvm().threads_attached(), 1);
-
-    drop(env);
```

Comment:
Why are you removing this?

Reply:
Because of https://travis-ci.org/github/jni-rs/jni-rs/jobs/729379085#L258

Reply:
By the way. What's a point to call drop here ? As I can see `JNIEnv` doesn't implement `Drop` trait.

Reply:
I see. Thanks for the clarification.

I'm thinking this is probably related to a recent change (#255), where `JNIEnv` now implements `Copy`.

Reply:
I'd consider clarifying the intent, as in the other method, e.g.,

```
    // Dropping JNIEnv obtained from a daemon attach has no effect on the thread status.
```